docs/fsharp/language-reference/records.md

```diff
@@ -116,11 +116,15 @@ Like union and structure types, records have structural equality semantics. Clas
 
 If you write the same code with classes, the two class objects would be unequal because the two values would represent two objects on the heap and only the addresses would be compared (unless the class type overrides the `System.Object.Equals` method).
 
+If you need class like comparison or rather reference-equality for records add the attribute `[<ReferenceEquality>]` above the record.
```

Comment:
Since it's been a month, I took care of the update, so we can merge. @cartermp can you please review it? Thanks!